frontend/src/openapi/models/executiveSummarySchemaProjectFlagTrendsItem.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Generated by Orval
+ * Do not edit manually.
+ * See `gen:api` script in package.json
+ */
+
+export type ExecutiveSummarySchemaProjectFlagTrendsItem = {
```

Comment:
I recommend running it again one more time to get health and time to production that got added a second ago :) We'll avoid another orval run in future